config/locales/en.yml

```diff
@@ -31,6 +31,8 @@ en:
     local_transaction:
       invalid_postcode: "Please enter a valid full UK postcode."
       no_local_authority_html: "Sorry, we can't find the local council for your postcode. Try using <a href='/find-your-local-council'>the local council directory</a>."
+      no_local_interaction: "We don't have a direct link to this service from %{local_authority_name}"
+      no_local_interaction_with_link: "We don't have a direct link to this service from %{local_authority_name}, but you could try the <a href=%{contact_url} rel='external'>%{local_authority_name} website</a>."
```

Comment:
If we named these with `_html` suffixes then we could drop the `.html_safe` bit when we draw them in the view.  See: http://guides.rubyonrails.org/i18n.html#using-safe-html-translations